src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -1232,6 +1232,21 @@ export class CreateVmAction extends React.Component {
                     </span>
                 </Tooltip>
             );
+        else
+            createButton = (
+                <Tooltip id={this.props.mode + '-button-tooltip'}
+                         position={this.props.mode === "create" ? "top-end" : "top"}
+                         className={this.props.mode === "create" && "custom-arrow"}
+                         content={this.props.mode === "create"
+                             ? _("Create VM machine from local or network installation medium")
```

Comment:
"VM machine" is redundant. Either "VM" or "virtual machine", but please keep it consistent with the other string below. (IMHO, "VM" in a tooltip is fine)

Reply:
Fixed